chapters/ajax/ajax_request_without_jquery.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: recipe
+title: Ajax Request Without jQuery
+chapter: Ajax
+---
+## Problem
+
+You want to load data from your server via AJAX without using the jQuery library.
+
+## Solution
+
+You will use the native <a href="http://en.wikipedia.org/wiki/XMLHttpRequest" target="_blank">XMLHttpRequest</a> object.
+
+Let's set up a simple test HTML page with a button.
+
+{% highlight html %}
+<!DOCTYPE HTML>
+<html lang="en-US">
+<head>
+	<meta charset="UTF-8">
+	<title>XMLHttpRequest Tester</title>
+</head>
+<body>
+	<h1>XMLHttpRequest Tester</h1>
+	<button id="loadDataButton">Load Data</button>
+	
+	<script type="text/javascript" src="XMLHttpRequest.js"></script>
+</body>
+</html>
+{% endhighlight %}
+
+When the button is clicked, we want to send an Ajax request to the server to retrieve some data.  For this sample, we have a small JSON file.
+
+{% highlight javascript %}
+// data.json
+{
+  message: "Hello World"
+}
+{% endhighlight %}
+
+Next, create the CoffeeScript file to hold the page logic.  The code in this file creates a function to be called when the Load Data button is clicked.
+
+{% highlight coffeescript linenos %}
```

Comment:
https://coffeescript-cookbook.github.io/chapters/ajax/ajax_request_without_jquery

```
# XMLHttpRequest.coffeeloadDataFromServer=->req=newXMLHttpRequest()req.addEventListener'readystatechange',->ifreq.readyStateis4# ReadyState CompletesuccessResultCodes=[200,304]ifreq.statusinsuccessResultCodesdata=eval'('+req.responseText+')'console.log'data message: ',data.messageelseconsole.log'Error loading data...'req.open'GET','data.json',falsereq.send()loadDataButton=document.getElementById'loadDataButton'loadDataButton.addEventListener'click',loadDataFromServer,false
```

U should remove `linenos `, as I understand